ui/src/components/Form/BorderButton/index.tsx

```diff
@@ -0,0 +1,37 @@
+import classNames from 'classnames/bind';
+import React from 'react';
+import { MdKeyboardArrowDown } from 'react-icons/md';
+
+import style from './style.module.scss';
+
+const c = classNames.bind(style);
+
+export interface ButtonProps {
+  /**
+   * Rander button content
+   */
+  children: React.ReactElement;
+  /**
+   * If user want to show down icon
+   */
+  hasNoUpArrow?: boolean;
```

Comment:
1. We should avoid negative flags. `hasNoUpArraow` is not any better than `hasUpArrow`, is two characters long, and adds more cognitive load to the reader
2. Prop says `up arrow`, but code below actually adds down arrow
3. `BorderButton` shouldn't hard-code what kind of arrow icon the component is going to have. I suggest we let the consumer of this component decide which (and if) icon they want to show. This component should only provide a button with border